packages/playground/src/weblets/micro_vm.vue

```diff
@@ -2,8 +2,8 @@
   <weblet-layout
     ref="layout"
     @mount="layoutMount"
-    :cpu="cpu"
-    :memory="memory"
+    :cpu="solution?.cpu"
+    :memory="solution?.memory"
     :disk="disks.reduce((total, disk) => total + disk.size, rootFsSize)"
```

Comment:
update the disk here to be consistence with the solution disk :
after logging  the filters i see the value is only 2 <the init value of `rootFsSize`
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/62248851/b705ca8f-1720-4501-a91e-707318a16903)
![Screenshot from 2023-07-24 18-08-58](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/62248851/3e6a7cdc-891a-4939-9759-7c80eb892fa7)